sycl/include/sycl/detail/pi.h

```diff
@@ -2063,6 +2063,17 @@ __SYCL_EXPORT pi_result piTearDown(void *PluginParameter);
 /// timestamp
 __SYCL_EXPORT pi_result piPluginGetLastError(char **message);
 
+/// API to get backend specific optimization option.
+/// \param opt_level is an integer that contains frontend optimization level.
+/// \param backend_option is used to return the backend optimization option
+/// corresponding to frontend optimization level.
```

Comment:
Please update the comment to not be optimization-level specific

Reply:
Done in upcoming commit. Thanks